python/triton/common/build.py

```diff
@@ -18,7 +18,7 @@ def is_hip():
 
 @functools.lru_cache()
 def libcuda_dirs():
-    libs = subprocess.check_output(["ldconfig", "-p"]).decode()
+    libs = subprocess.check_output(["/sbin/ldconfig", "-p"]).decode()
```

Comment:
Does it have to be `/sbin/ldconfig` instead of `ldconfig`?

Maybe you could add something in `msg` like "make sure ldconfig is configured in $PATH?" I understand that generally `ldconfig` is under `/sbin` or `/usr/sbin` but not sure if the location varies based on distributions. 

It looks good to me otherwise.

Reply:
Got you. LGTM

Reply:
I also checked the Unix file system standard, `/sbin` should always exist for any Unix distributions. https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard

Reply:
@ptillet I think we could go merge the PR